connectors/kafka/element-templates/kafka-inbound-connector-intermediate.json

```diff
@@ -0,0 +1,266 @@
+    "eventDefinition": "bpmn:MessageEventDefinition"
+  },
+  "groups": [
+    {
+      "id": "authentication",
+      "label": "Authentication"
+    },
+    {
+      "id": "kafka",
+      "label": "Kafka"
+    },
+    {
+      "id": "activation",
+      "label": "Activation"
+    },
+    {
+      "id": "variable-mapping",
+      "label": "Variable Mapping"
+    }
+  ],
+  "properties": [
+    {
+      "type": "Hidden",
+      "value": "io.camunda:connector-kafka-inbound:1",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.type"
+      }
+    },
+    {
+      "type": "Hidden",
+      "generatedValue": {
+        "type": "uuid"
+      },
+      "binding": {
+        "type": "bpmn:Message#property",
+        "name": "name"
+      }
+    },
+    {
+      "label": "Authentication type",
+      "description": "Username/password or custom",
+      "id": "authenticationType",
+      "group": "authentication",
+      "type": "Dropdown",
+      "value": "credentials",
+      "choices": [
+        {
+          "name": "Credentials",
+          "value": "credentials"
+        },
+        {
+          "name": "Custom",
+          "value": "custom"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:property",
+        "name": "authenticationType"
+      }
+    },
+    {
+      "label": "Username",
+      "description": "Provide the username (must have permissions to produce message to the topic)",
+      "group": "authentication",
+      "type": "String",
+      "feel":"optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:property",
+        "name": "authentication.username"
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "credentials"
+      }
+    },
+    {
+      "label": "Password",
+      "description": "Provide a password for the user",
+      "group": "authentication",
+      "type": "String",
+      "feel":"optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:property",
+        "name": "authentication.password"
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "credentials"
+      }
+    },
+    {
+      "label": "Bootstrap servers",
+      "description": "Provide bootstrap server(s), comma-delimited if there are multiple",
+      "group": "kafka",
+      "type": "String",
+      "feel":"optional",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "topic.bootstrapServers"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Topic",
+      "description": "Provide the topic name",
+      "group": "kafka",
+      "type": "String",
+      "feel":"optional",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "topic.topicName"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Additional properties",
```

Comment:
Makes sense to have `Additional properties` as the last field, but I would still keep it inside the Kafka group, so I would place right after `Auto offset reset`.

Reply:
I will leave it as is for now, because this is how it is for the Kafka inbound start event template, might be confusing to have different ordering for the intermediate.